### ISSUE TITLE
[BUG] Broken links in plugin README files

### ISSUE THREAD

**Issue by AWSHurneyt** [opensearch-project/opensearch-plugins#163]
### What is the bug?
Several of the plugins (e.g., [anomaly detection](https://github.com/opensearch-project/anomaly-detection/blob/main/README.md?plain=1#L4), [alerting](https://github.com/opensearch-project/alerting/blob/main/README.md?plain=1#L4), [ISM](https://github.com/opensearch-project/index-management/blob/main/README.md?plain=1#L5), etc.) contain links to the opendistro website in the READMEs for their back **and** frontend repositories. These links redirect to [opensearch.org](opensearch.org), but not to a functioning forum page. The link in the [alerting dashboards README](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/README.md?plain=1#L5) for example links to https://forum.opensearch.org/c/Use-this-category-for-all-questions-around-machine-learning-plugins, which displays a `Oops! That page doesn’t exist or is private` error message.

This issue replaces the broken links in many of the READMEs, but frontend plugins in particular are not addressed by it.
https://github.com/opensearch-project/documentation-website/issues/921

As there are several plugins with READMEs that contain these broken links, it may be worthwhile to campaign for having each plugin replace that link with an updated one.

### How can one reproduce the bug?
Visit the README file for the plugins, and click the link to the opendistro forum.

### What is the expected behavior?
These links should take the user to the appropriate forum on the OpenSearch website.

### What is your host/environment?
MacOS 12.5.1

### Do you have any screenshots?
_If applicable, add screenshots to help explain your problem._

### Do you have any additional context?
_Add any other context about the problem._

**Comment by saratvemulapalli** [comment]
Thanks @AWSHurneyt, @cwillum for opening this up. 
I understand the Readme's are going out of date and links are broken. 
Its nice to fix them, but I am afraid they will be lost again. 

We've build link checker[1] for this repository which mandates every PR to update broken links. 
We've already updated the documentation[2]. 
Could you use `meta`[3] to open up issues in all plugins to implement broken links checker and fix the readme's along with it? We can make this as a campaign and label it for a release.

[1] https://github.com/opensearch-project/opensearch-plugins/blob/main/.github/workflows/links.yml 
[2] https://github.com/opensearch-project/opensearch-plugins/blob/main/STANDARDS.md#broken-links-checker
[3] https://github.com/opensearch-project/opensearch-plugins/blob/main/.meta

**Comment by saratvemulapalli** [comment]
Coming back, @AWSHurneyt are you open to taking care of this and run a campaign ?

**Comment by AWSHurneyt** [comment]
@saratvemulapalli I'm currently committed to a feature we're looking to launch for 2.4, but I should have some bandwidth after that release. Is there an SOP for running a campaign?

**Comment by dblock** [comment]
> @saratvemulapalli I'm currently committed to a feature we're looking to launch for 2.4, but I should have some bandwidth after that release. Is there an SOP for running a campaign?

No, but we should! I opened https://github.com/opensearch-project/project-meta/issues/61, want to create a campaign for above and document the steps?
